src/hamster-cli.py

```diff
@@ -46,7 +46,7 @@
 from hamster.lib import default_logger, stuff
 from hamster.lib import datetime as dt
 from hamster.lib.fact import Fact
-
+from hamster.lib.notifsmanager import notifs_mgr
```

Comment:
`notifs_mgr` seems to be referenced nowhere in this file. What am I missing?

Reply:
Must be remnants of my debugging session. I'll remove it, thanks!